### PR TITLE
Issue #000 fix: Add footer to site

### PR DESCRIPTION
**Pull request by lakhanmandloi**
PFA screenshot of footer -
footer

### CODE REVIEW

**Comment by harishgilimi** [comment]
@lakhanmandloi  let me know when can i merge this PR

**Comment by lakhanmandloi** [comment]
@harishgilimi - HTML CSS wise this is good to merge and functionally it is working correctly. However, from the Angular & Js implementation perspective can you pls review it if this is implemented in the correct way? If it is good from the Angular and Js perspective then pls merge it. I will notify the managers for deployment request. If the implementation is the wrong way then I will ask pm to push it in next release.

File: src/app/client/src/app/modules/core/components/main-footer/main-footer.component.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+          Contact for queries:
+        </label>
+        <a class="d-block mb-30" href="mailto:diksha-support@ncte-India.org">
+          diksha-support@ncte-india.org
+        </a>
+        <label class="d-block">
+          <a href="/term-of-use.html">
+            Terms of Use
+          </a>
+          <span>| &copy; {{date | date:'yyyy'}}, DIKSHA</span>
```
Comment (harishgilimi) [inline src/app/client/src/app/modules/core/components/main-footer/main-footer.component.html]: copy right update is dynamic check with namrita

**Comment by lakhanmandloi** [comment]
@harishgilimi  the circleci test-coverage is passed but seems like ```npm install -g codacy-coverage && cat ./coverage/lcov.info | codacy-coverage``` is failing. Some has committed code in the repo.

**Pull request merged**